common4j/src/main/com/microsoft/identity/common/java/cache/SharedPreferencesAccountCredentialCacheWithMemoryCache.java

```diff
@@ -86,6 +88,7 @@ private void load() {
 
         synchronized (mCacheLock) {
             try {
+                mSha1Cleared = loadSha1ClearedFlag();
```

Comment:
>mSha1Cleared = loadSha1ClearedFlag(); [](http://example.com/codeflow?start=16&length=37)

to try to prevent keeping more state around in this class that needs reasoning about, could this move to a local boolean, and a parameter 'boolean cleanUpSha1AccessTokens' on `loadCredentialWithKeys`?